apps/chat/src/components/Chat/Publish/ReviewApplicationDialogView.tsx

```diff
@@ -106,7 +106,7 @@ export function ReviewApplicationDialogView() {
         {application?.maxInputAttachments !== undefined && (
           <div className="flex gap-4">
             <span className="w-[122px] text-secondary">
-              {t('Max. attachments number:')}
+              {t(' Max. attachments number:')}
```

Comment:
```suggestion
              {t('Max. attachments number: ')}
```

I suppose something like this?